ScintillaNet_FindReplaceDialog/FindAllResults/FindAllResultsPanel.cs

```diff
@@ -85,10 +95,9 @@ public void UpdateFindAllResults(FindReplace FindReplace, List<ScintillaNET_Find
 
                 if (startLine == endLine)
                 {
-                    string resultsLinePrefix = string.Format(Properties.Resources.FindAllResults_ResultsLinePrefix, startLine + 1);
+	                string resultsLinePrefix = string.Format("Line {0}: ", startLine + _startingLineNumber);
```

Comment:
Removing this breaks the localization support as this is also localized in German, so either:
* Change the resource files to accept two parameters
* Leave this as it was before
![image](https://github.com/VPKSoft/ScintillaNET-FindReplaceDialog/assets/40712699/237c836f-dbb3-4fcd-bfb4-edd600d76327)

Reply:
I've rolled this back, but again I'm new to git and GitHub so I'm not sure if I've added correctly.

Reply:
The rollback might have failed? Your fork doesn't contain the reverted version either: https://github.com/olaws/ScintillaNET-FindReplaceDialog/blob/869340f66eba62de7da99c8adaaf4948c996cb9c/ScintillaNet_FindReplaceDialog/FindAllResults/FindAllResultsPanel.cs#L113. Perhaps you forgot to push the change?

Reply:
This looks good now also.